ios/UnflowMetadataBridge.swift

```diff
@@ -20,11 +20,20 @@ struct UnflowMetadataBridge {
         if let analyticsValue = value as? UnflowAnalyticsValue {
             return analyticsValue
         } else if let numberValue = value as? NSNumber {
-            return numberValue.doubleValue
+            if CFGetTypeID(numberValue) == CFBooleanGetTypeID() {
```

Comment:
@SwiftyAlex I noticed that we supported boolean types natively but not in the converter. This seems to work but not sure if it's correct